plugins/woocommerce/tests/performance/tests/gh-action-pr-requests.js

```diff
@@ -22,7 +22,7 @@ import { ordersFilter } from '../requests/merchant/orders-filter.js';
 import { addOrder } from '../requests/merchant/add-order.js';
 import { ordersAPI } from '../requests/api/orders.js';
 import { homeWCAdmin } from '../requests/merchant/home-wc-admin.js';
-import { useCartCheckoutShortcodes } from '../setup/cart-checkout-shortcode.js';
+import { setCartCheckoutShortcodes } from '../setup/cart-checkout-shortcode.js';
```

Comment:
this is to avoid linting error from React Hook asking us to change the function starting with use or something about big first letter. This fix is as simple as it is.